apps/freelance/src/pages/TalentListPage/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { ReactElement, useState } from 'react';
+import { Avatar, Col, Input, List, Row } from 'antd';
+import { useTranslation } from 'react-i18next';
+import { UserOutlined } from '@ant-design/icons';
+import { SmallCard } from '@freelance/components';
+import { useGetFreelancerQuery } from 'redux/services/freelancers';
+
+import { User } from './model';
+import { StyledPagination } from './styles';
+import {
+  Container,
+  SmallCardContainer,
+  StyledCard,
+  StyledCardHeader,
+  StyledHeader,
+  StyledInformation,
+  StyledName,
+} from './styles';
+
+const TalentListPage = (): ReactElement => {
+  const { Search } = Input;
+  const [search, setSearch] = useState<string>('');
+  const { t } = useTranslation();
+  const [page, setPage] = useState<number>(1);
+  const { data } = useGetFreelancerQuery(page);
```

Comment:
its mess state should be near state, hook, near hook, constants near constants

Reply:
fixed